WalletWasabi.Gui/Controls/WalletExplorer/CoinListViewModel.cs

```diff
@@ -72,6 +73,7 @@ public class CoinListViewModel : ViewModelBase
 		public ReadOnlyObservableCollection<CoinViewModel> Coins => _coinViewModels;
 
 		public bool DisplayCommonOwnershipWarning { get; set; } = false;
+		public WalletService WalletService { get; }
```

Comment:
no reason to have this public, hence im my PR private field, we dont want to encourage people to accidently depend on this vm by accessing this, it should be an implementation detail not an api.

Reply:
ACK.